README.md

```diff
@@ -46,9 +46,12 @@ for information on how we do:
 
 - Testing
 - Pull Requests
-- Code Style
 - Documentation
 
+We use [pre-commit](https://pre-commit.com/) to maintain consistent code style. Once
+installed, it will run automatically with every commit. You can also run it manually
+with `make lint`.
```

Comment:
Recommend including the --no-verify flag as a workaround.

Reply:
Added! Look OK?